.evergreen/run-mongodb-aws-test.sh

```diff
@@ -35,4 +35,5 @@ set -x
 
 echo "Running tests with ${JDK}"
 ./gradlew -version
-./gradlew -PjdkHome=/opt/java/${JDK} -Dorg.mongodb.test.uri=${MONGODB_URI} --stacktrace --debug --info driver-core:test --tests AwsAuthenticationSpecification
+./gradlew -PjdkHome=/opt/java/${JDK} -Dorg.mongodb.test.uri=${MONGODB_URI} --stacktrace --debug --info --no-build-cache \
+driver-core:cleanTest driver-core:test --tests AwsAuthenticationSpecification
```

Comment:
Necessary because the evergreen config calls this script multiple times in successive functions, with different URIs, and without some sort of clean here all but the first call becomes a no-op.  This isn't a new problem, but I noticed it when debugging this new feature.

Reply:
Good idea. Done.